integrations/dell_idrac/dell_idrac.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+from kdnrm.saas_plugins import SaasPluginBase
+from kdnrm.saas_type import ReturnCustomField, Secret, SaasUser, SaasConfigItem
+from kdnrm.exceptions import SaasException
+from typing import List, TYPE_CHECKING
+from kdnrm.log import Log
+
+try:
+    import requests
+except ImportError:
+    raise SaasException('Missing required module: requests. Please install it using "pip install requests"')
```

Comment:
While raising the exception, we are hiding the actual exception. Please consider following suggestion:

except ImportError as exc:
    raise SaasException('Missing required module: requests. Please install it using "pip install requests"') from exc